src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java

```diff
@@ -9,10 +9,13 @@ class InputCamelCaseDefined {
   boolean supportsIpv6OnIos;
 
   void XmlHttpRequest() {}
+  // violation above 'Method name 'XmlHttpRequest' must match pattern'
 
   void YouTubeImporter() {}
+  // violation above 'Method name 'YouTubeImporter' must match pattern'
 
   void YoutubeImporter() {}
+  // violation above 'Method name 'YoutubeImporter' must match pattern'
```

Comment:
kept these extra violations intentionally, it was to show that these method names are not part of camel case section.

Reply:
what check does this violation ?
not clear why there is violation here

Reply:
oh I got it .... name should be started from lowercase character